django/db/models/functions/datetime.py

```diff
@@ -223,6 +223,11 @@ def as_postgresql(self, compiler, connection, **extra_context):
             compiler, connection, template="STATEMENT_TIMESTAMP()", **extra_context
         )
 
+    def as_mysql(self, compiler, connection, **extra_context):
+        return self.as_sql(
+            compiler, connection, template="CURRENT_TIMESTAMP(6)", **extra_context
+        )
```

Comment:
Seems to be from [this discussion](https://github.com/django/django/pull/13709#discussion_r528652652)

This change is ripe for cherry-picking into a separate PR, and changelog note would be good.

Reply:
https://github.com/django/django/pull/16111